### ISSUE TITLE
Add "I'm not the right person" to request options for authors

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/discussion#202]
Often corresponding authors will get asked for stuff that other authors on the paper don't have. It would be nice to give them a simple option to forward the request onto another person (in a way we can track).

**Comment by markmacgillivray** [comment]
Note to self - this should just be text added to the author request email, saying if they are the wrong person please sign up to oabutton then go to the request page and change the contact email address for the article.

So just need to add a check on the request page to allow current user to say "I was requested this but it is not me" - then they give a new email address, and we save the old email address as one not to try for this article again.

This encourages authors to sign up too, and take a direct interest in requests - even if it just starts as them letting us know they can't help with a particular request, we then later may get them contributing to other requests.

**Comment by JosephMcArthur** [comment]
Assigned @chealsye for some copy on this

**Comment by markmacgillivray** [comment]
@JosephMcArthur @chealsye note that things like this would be greatly impacted by anonymisation too - if we don't have user accounts, we cannot know who we should let edit a request. Either nobody can, or everybody can.

Having accounts but allowing anonymous requests would enable us to have this extra level of functionality, but would still mean we could not identify the user that created a request so could not allow them to improve their request content after they created it.

Authors will still have to sign in to be able to tell us they are the wrong person and to provide us an alternative email address. Without user accounts, we could still record that we had contacted the wrong person, but would have a problem knowing who we should accept new emails from.

Essentially, allowing anyone to change the email address we contact would leave the system vulnerable to abuse. So, without accounts, we can't have this kind of functionality.

I know that anonymity was not specifically related to getting rid of accounts, but as it has come up in discussion, it is worth considering here too. When I suggested that we want to drive authors towards signing up when telling us they are not the right person, I was assuming that our aim IS to get people to sign up and therefore to be open about who they are, their use of the service, and therefore implicitly their support for the concept of open access in research. If we decide to move away from that philosophy by introducing anonymity at certain points, then we have to also consider if it is right to want to drive authors (or anyone else) to sign up at all.
